src/lando/api/legacy/api/stacks.py

```diff
@@ -67,13 +60,13 @@ def get(phab: PhabricatorClient, revision_id: str):
     )
     revision = phab.single(revision, "data", none_when_empty=True)
     if revision is None:
-        return not_found_problem
+        raise Http404
```

Comment:
We should add some parameters so it's more obvious what is missing, ie `raise Http404("The requested revision does not exist")`

Reply:
I am re-thinking the need to have a separate "revision not found" functionality vs. just a standard 404. I'm not sure if adds any value, and it also simplifies the implementation of the "fake" 404 we provide when a user is not authorized to view a revision. I don't see the value in having custom 404s depending on what type of page you are trying to access. If this was the API I think it could be useful to have a more specific error, but in this case Django is going to interpret this as a standard 404 no matter what string we pass here. Thoughts?

Reply:
According to [the docs](https://docs.djangoproject.com/en/5.0/topics/http/views/#the-http404-exception) the error message only appears in debug mode, so I think we ought to include it.

Reply:
Fair enough, it certainly doesn't hurt.